inc/define.php

```diff
@@ -571,6 +579,7 @@
    Printer::getType()            => "pics/impact/printer.png",
    Rack::getType()               => "pics/impact/rack.png",
    Software::getType()           => "pics/impact/software.png",
+   Database::getType()           => "",
```

Comment:
TODO

Reply:
Yes, but I do not have any idea where to find an image.

Reply:
ping @orthagh @AdrienClairembault

Reply:
Our source for icons is https://www.flaticon.com/authors/smashicons (it's in pics/impact/README.txt if you forget the url).

For a database, the "classic" icon seem to be this one : https://www.flaticon.com/free-icon/database_148825